docs/src/components/Tabs.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import PropType from 'prop-types';
+
+export const TabItem = (props) => {
+	return props.children;
+};
+TabItem.propTypes = {
+	children: PropType.oneOfType([PropType.string, PropType.array]),
+};
+
+export const Tabs = (props) => {
+	const data = props.children || [];
+	const [visibleTab, setVisibleTab] = useState(0);
+	const listTitles = data.map((item, index) => (
+		<button
+			aria-controls={`tab__panel-${item.props.title}`}
```

Comment:
Problem here I just thought about.  need to create a separate attribute for the title and id fields.  If a title like "Vanilla JS" is passed in, things start breaking.

Reply:
using label for the second param (anything that is not display)